packages/actor-query-source-identify-rdfjs/lib/QuerySourceRdfJs.ts

```diff
@@ -78,8 +83,8 @@ export class QuerySourceRdfJs implements IQuerySource {
 
     // Get bindings directly if the source allows it
     // This will be more efficient, as it avoids the intermediary quads translation and representation.
-    if (this.source.matchBindings) {
-      const rawStream = this.source.matchBindings(
+    if ((<any> this.source).matchBindings) {
```

Comment:
I suspect this any cast (and all of the other ones below) may not be necessary when replacing this check with `'matchBindings' in this.source`. Can you try?